### PR TITLE
qualcommax: backport more changes for ipq6018 and ipq8074

### PR DESCRIPTION
**Pull request by aiamadeus**
- Mark patches as upstream
 - Backport more upstream changes
 - Handle conflicts and refresh patches

The cpufreq and pmic for ipq60xx need more rework.

### CODE REVIEW

File: target/linux/qualcommax/patches-6.1/0061-v6.9-phy-qcom-qmp-usb-fix-serdes-init-sequence-for-IPQ6018.patch
Comment (mantas-p) [inline target/linux/qualcommax/patches-6.1/0061-v6.9-phy-qcom-qmp-usb-fix-serdes-init-sequence-for-IPQ6018.patch:1]: this one is actually in v6.8

**Comment by robimarko** [comment]
Thanks for doing this, has this been runtime tested?

**Comment by aiamadeus** [comment]
> Thanks for doing this, has this been runtime tested?

I tested this on ipq8072 (aw1000) and ipq6000 (gl-ax1800).

**Comment by robimarko** [comment]
Thanks! Rebased on top of main and merged!

**Pull request merged**